src/components/Members.tsx

```diff
@@ -3,17 +3,21 @@ import styles from './Members.module.css';
 
 import MemberCard from "./MemberCard";
 
-type Member = {
+export type Member = {
+  // should implement a few custom types later
   readonly name: string;
-  readonly image: string;
-  readonly website: string;
-  readonly github: string;
-  readonly linkedin: string;
-  readonly bio: string;
-  readonly favoredLink: string;
+  readonly bio?: string; // should be shorter than 120 characters
+  readonly image?: string;
+  readonly website?: string;
+  readonly github?: string;
+  readonly linkedin?: string;
+  readonly email?: string; // should begin with "mailto:"
+  readonly favoredLink?: "website" | "github" | "linkedin" | "email";
 };
```

Comment:
I'm thinking about making these types more granular 

For example:
- `email` must begin with `mailto:` followed by the email address
- `bio` should be a string no longer than 120 characters
- `linkedin` and `github` should begin with their domain

Reply:
I'm still going through the TypeScript Manuals for ideas
Let's hope I do find something helpful

Reply:
> I'm thinking about making these types more granular 
> 
> 
> 
> For example:
> 
> - `email` must begin with `mailto:` followed by the email address
> 
> - `bio` should be a string no longer than 120 characters
> 
> - `linkedin` and `github` should begin with their domain

This is impossible to achieve in typescript

Reply:
> > I'm thinking about making these types more granular 
> > 
> > 
> > 
> > For example:
> > 
> > - `email` must begin with `mailto:` followed by the email address
> > 
> > - `bio` should be a string no longer than 120 characters
> > 
> > - `linkedin` and `github` should begin with their domain
> 
> This is impossible to achieve in typescript 

That's sad : (
I was just reading through this issue https://github.com/microsoft/TypeScript/issues/6579 and thought if there exists a workaround for now

Reply:
I'll see if i can find a solution by tomorrow. 

Otherwise we'll have to leave it like this. Implementing a format check during run-time would be unnecessary (plus unwise).

Reply:
> I'll see if i can find a solution by tomorrow. 
> 
> Otherwise we'll have to leave it like this. Implementing a format check during run-time would be unnecessary (plus unwise).

In general that will require a dependent type system, and that's usually not what most of the programming language can provide.

Reply:
> 
> In general that will require a dependent type system, and that's usually not what most of the programming language can provide.

Thanks for pointing that out! I'm still pretty new to these PL concepts and will definitely look more into things related with type systems.